routers/api/packages/rpm/rpm.go

```diff
@@ -85,6 +88,7 @@ func GetRepositoryFile(ctx *context.Context) {
 }
 
 func UploadPackageFile(ctx *context.Context) {
+	distribution := ctx.Params("distribution")
```

Comment:
Here you could move it above `CreatePackageOrAddFileToExisting`. Then initialization and usage are at the same place.